Profiler/CProfilerCallback.cpp

```diff
@@ -43,7 +43,24 @@ CProfilerCallback::~CProfilerCallback() {
 	DeleteCriticalSection(&criticalSection);
 }
 
+/** Whether the given value ends with the given suffix. */
+inline bool endsWith(std::string const & value, std::string const & suffix)
+{
+	if (suffix.size() > value.size()) return false;
+	return std::equal(suffix.rbegin(), suffix.rend(), value.rbegin());
+}
+
 HRESULT CProfilerCallback::Initialize(IUnknown* pICorProfilerInfoUnkown) {
+	std::string process = getProcessInfo();
+	std::string processToProfile = getEnvironmentVariable("PROCESS");

```

Comment:
Maybe rename getEnvironmentVariable to getConfigValue?

Reply:
In order to distinguish it from the config options that can also come from the config file, we need to be a bit more specific, but rename was a good idea, since I struggled last time with the fact that it assumes the COR_PROFILER_ prefix.